style/README.md

```diff
@@ -82,25 +82,26 @@ Sass
 
 ### Formatting
 
-* Use the *Scss* syntax.
+* Use the SCSS syntax.
 * Use hyphens when naming mixins, extends, classes, functions & variables: `span-columns` not `span_columns` or `spanColumns`.
-* Use space between property and value: `width: 20px` not `width:20px`.
-* Use a blank line above selector that has styles.
-* Prefer hex color codes `#000`.
+* Use one space between property and value: `width: 20px` not `width:20px`.
+* Use a blank line above a selector that has styles.
+* Prefer hex color codes `#fff`.
 * Use `//` for comment blocks not `/* */`.
-* Use a space between selector and `{`.
+* Use one space between selector and `{`.
 * Use double quotation marks.
 * Use only lowercase, including colors.
-* Don't add a unit specification after `0` values, unless required by a mixin.
+* Don’t add a unit specification after `0` values, unless required by a mixin.
```

Comment:
Want to open a pull request for Markdown and smart quotes? (Or something similar)